README.md

```diff
@@ -11,6 +11,7 @@
 
 ## Live Demo
 
+Deploy to Github pages
```

Comment:
[OPTIONAL] You can create links in markdown files with the following sintax [alt txt] (URL) <-- (Remove the space between brackets and parenthesis) 

- [Sample](https://koomekevo.github.io/kevin-koome-portfolio/)